src/commcare_cloud/ansible/roles/commcarehq/templates/localsettings.py.j2

```diff
@@ -577,8 +577,7 @@ INDICATOR_NAMESPACES = {
         ('mvp_indicators', "MVP"),
     ),
 }
-# should be 'ok' if couchdb version < 1.1.0, or cloudant, else 'update_after'
-COUCH_STALE_QUERY = '{{ localsettings.COUCH_STALE_QUERY | default('update_after') }}'
+COUCH_STALE_QUERY = 'update_after'
```

Comment:
I think this is also the default: https://github.com/dimagi/commcare-hq/blob/5aca7bd9d95b0805fec61417df6ffd65c618f252/settings.py#L735 FWIW. What do you think about just removing this line entirely?

Reply:
I'm not opposed to it. The main reason I didn't was that it would introduce a change to all environments localsettings which is a pain to deploy and introduces a bit of overhead. As it is now the PR only needs to be deployed to one environment.